### ISSUE TITLE
Show the email in the chip of the customers

### ISSUE THREAD

**Issue by giosueDelgado** [vendure-ecommerce/vendure#2850]
**Is your feature request related to a problem? Please describe.**
We have received several inquiries from clients asking if it is possible to include the customer's email directly within the chip of the email. This query arises because there is a need to streamline the process of accessing customer email information directly, rather than navigating through multiple steps.

**Describe the solution you'd like**
To address this, a flexible solution could be implemented in the form of a global configuration setting. This setting would allow administrators to opt-in to displaying the customer's email directly within the chip of the email for all users. This would centralize and simplify the visibility of customer emails, making the information readily accessible without additional navigation.

**Describe alternatives you've considered**
An alternative to embedding the email in the chip would be to display the customer's email in a specific column within the relevant table. This would also simplify access but would keep the information in a traditional tabular context rather than integrating it into the chip interface. 

**Additional context**
Currently, users spend a significant amount of time opening each request individually to view customer emails especially if is guest. By streamlining this process, we can enhance user efficiency and satisfaction. Integrating email information directly into the email chip or displaying it prominently within a table would significantly reduce the time and steps needed to access this information.

**Comment by oidt** [comment]
Is it possible to add some screenshots about your feature request?

**Comment by giosueDelgado** [comment]
Sure.
Here an example:

![image](https://github.com/vendure-ecommerce/vendure/assets/11545723/0a162284-0c40-428b-be06-d030e1ee4e5b)

**Comment by dlhck** [comment]
@giosueDelgado I don't think it is smart to include it in the chip as it does not look great. I think we could combine a chip with a tooltip. We could even add a `copy` action to the tooltip, that copies the email to the clipboard.

**Comment by dlhck** [comment]
Could be done together with #2903
